core/pom.xml

```diff
@@ -153,8 +143,12 @@
 			<artifactId>opennlp-tools</artifactId>
 		</dependency>
 		<dependency>
+			<!-- TODO: NemexWekaClassificationEDATest throws "java.lang.Exception: liblinear classes not in CLASSPATH!".
+            This requires a wrapper but the one in https://github.com/bwaldvogel/liblinear-weka is outdated. -->
 			<groupId>de.bwaldvogel</groupId>
 			<artifactId>liblinear</artifactId>
+			<version>2.11</version>
```

Comment:
@reckart when you say "by us", do you mean by DKPro? I just tried to build with 1.7.1 but it looks like it's incompatible with DKPro core 1.4.0:
```
java.lang.NoClassDefFoundError: Lde/bwaldvogel/liblinear/SolverType;
at java.lang.Class.getDeclaredFields0(Native Method)
...
at de.tudarmstadt.ukp.dkpro.core.maltparser.MaltParser.process(MaltParser.java:274)
...
```
Do you know which liblinear version is compatible with DKPro 1.4.0? And actually I'm trying to upgrade to DKPro 1.8.0, which I assume will work with the latest liblinear version, right?

If the problem with weka/liblinear is in the EOP code then it shouldn't be too difficult to refactor it to work with the latest version. So I'd rather try to keep the dkpro dependencies "happy" for now, so to speak.